security/certificate_types_descriptions/olm-certificates.adoc

```diff
@@ -9,10 +9,6 @@ toc::[]
 
 All certificates for OpenShift Lifecycle Manager (OLM) components (`olm-operator`, `catalog-operator`, `packageserver`, and `marketplace-operator`) are managed by the system.
 
-Operators installed via OLM can have certificates generated for them if they are providing API services. `packageserver` is one example.
+Operators installed using OLM can have certificates generated for them if they are providing API services, such as `packageserver`. Certificates in the `openshift-operator-lifecycle-manager` namespace are managed by OLM.
```

Comment:
Feel free to restructure the sentence below, but I think it captures the intent slightly better.

```suggestion
When installing operators that include webhooks or API Services in their CSV, OLM will create and rotate certificates for these resources. 
```

> Certificates in the `openshift-operator-lifecycle-manager` namespace are managed by OLM.

Could you clarify what you mean by "Certificates" in the  `openshift-operator-lifecycle-manager` namespace? Certificates are not a Kubernetes resource.

Reply:
Not sure the original intent here, but I assumed more like "certificates for resources in the namespace." @ahardin-rh ?